tests/scripts/run-xunit-perf.cmd

```diff
@@ -2,193 +2,332 @@
+  )
+  IF /I [%~1] == [-testEnv] (
+    set TEST_ENV=%~2
+    shift
+    shift
+    goto :parse_command_line_arguments
+  )
+  IF /I [%~1] == [-configuration] (
+    set TEST_CONFIG=%~2
+    shift
+    shift
+    goto :parse_command_line_arguments
+  )
+
+  if /I [%~1] == [-?] (
+    call :USAGE
+    exit /b 0
+  )
+  if /I [%~1] == [-help] (
+    call :USAGE
+    exit /b 0
+  )
+  if [%CORECLR_PERF%] == [] (
+    call :USAGE
+  )
+
+  exit /b %ERRORLEVEL%
+
+:set_test_architecture
+rem ****************************************************************************
+rem   Sets the test architecture.
+rem ****************************************************************************
+  IF /I [%TEST_ARCHITECTURE%] == [x86jit32] (
+      set TEST_ARCH=x86
+  ) ELSE (
+      set TEST_ARCH=%TEST_ARCHITECTURE%
+  )
+  exit /b 0
+
+:verify_core_overlay
+rem ****************************************************************************
+rem   Verify that the Core_Root folder exist.
+rem ****************************************************************************
+  set CORECLR_OVERLAY=%CORECLR_REPO%\bin\tests\Windows_NT.%TEST_ARCH%.%TEST_CONFIG%\Tests\Core_Root
+  if NOT EXIST "%CORECLR_OVERLAY%" (
+    call :print_error Can't find test overlay directory '%CORECLR_OVERLAY%'. Please build and run Release CoreCLR tests.
+    exit /B 1
+  )
+  exit /b 0
+
+:set_perf_run_log
+rem ****************************************************************************
+rem   Sets the script's output log file.
+rem ****************************************************************************
+  if NOT EXIST "%CORECLR_REPO%\bin\Logs" (
+    call :print_error Cannot find the Logs folder '%CORECLR_REPO%\bin\Logs'.
+    exit /b 1
+  )
+  set RUNLOG=%CORECLR_REPO%\bin\Logs\perfrun.log
+  exit /b 0
+
+:setup_sandbox
+rem ****************************************************************************
+rem   Creates the sandbox folder used by the script to copy binaries locally,
+rem   and execute benchmarks.
+rem ****************************************************************************
+  if exist sandbox rd /s /q sandbox
+  if exist sandbox call :print_error Failed to remove the sandbox folder& exit /b 1
+  if not exist sandbox mkdir sandbox
+  if not exist sandbox call :print_error Failed to create the sandbox folder& exit /b 1
+  cd sandbox
+  exit /b 0
+
+:generate_results_for_benchview
+rem ****************************************************************************
+rem   Generates results for BenchView, by appending new data to the existing
+rem   measurement.json file.
+rem ****************************************************************************
+  set BENCHVIEW_MEASUREMENT_PARSER=xunit
+  if defined IS_SCENARIO_TEST set BENCHVIEW_MEASUREMENT_PARSER=xunitscenario
+
+  set LV_MEASUREMENT_ARGS=
+  set LV_MEASUREMENT_ARGS=%LV_MEASUREMENT_ARGS% %BENCHVIEW_MEASUREMENT_PARSER%
+  set LV_MEASUREMENT_ARGS=%LV_MEASUREMENT_ARGS% "Perf-%BENCHNAME%.xml"
```

Comment:
@jorive can you please generalize this step, so that it works with scenario benchmarks that run multiple scenarios? 
For example, the LinkBench runs 5 scenarios and generates 5 different .xml result files.
So, I think getting all the .xml files is better than looking for perf-benchmark.xml